tests/cases/FileUploadExtension.phpt

```diff
@@ -0,0 +1,208 @@
+<?php declare(strict_types = 1);
+
+namespace Tests\Cases;
+
+use Nette\DI\Compiler;
+use Nette\DI\InvalidConfigurationException;
+use Nette\InvalidStateException;
+use Nette\UnexpectedValueException;
+use Tester\Assert;
+use Tester\TestCase;
+use Tests\Fixtures\Filter\InvalidFilter;
+use Tests\Fixtures\Filter\ValidFilter;
+use Tests\Fixtures\Model\InvalidUploadModel;
+use Tests\Fixtures\Model\ValidUploadModel;
+use Tests\Fixtures\Renderer\InvalidRenderer;
+use Tests\Toolkit\Container;
+use Tests\Toolkit\Helpers;
+use Zet\FileUpload\Exception\InvalidArgumentException;
+use Zet\FileUpload\Filter\IMimeTypeFilter;
+use Zet\FileUpload\Model\IUploadModel;
+use Zet\FileUpload\Template\Renderer\Bootstrap4Renderer;
+use Zet\FileUpload\Template\Renderer\IUploadRenderer;
+
+require_once __DIR__ . '/../bootstrap.php';
+
+/**
+ * @testCase
+ */
+final class FileUploadExtension extends TestCase
+{
+
+	public function testConfig(): void
+	{
+		Assert::noError(function (): void {
+			Container::of()
+				->withDefaults()
+				->withCompiler(function (Compiler $compiler): void {
+					$compiler->addConfig(Helpers::neon('
```

Comment:
Pouzil by jsi prosim heredoc/nowdoc?

```php
$compiler->addConfig(Helpers::neon(<<<NEON
						fileUpload:
							maxFiles: 10
							maxFileSize: 2M
							fileFilter: Zet\FileUpload\Filter\ImageFilter
						NEON));
```

Reply:
Heredoc mi neprojde přes QA - viz https://github.com/hermajan/jQuery-FileUpload/runs/3735574887?check_suite_focus=true. Je to třeba nějak jinak upravit?